include/rtc/h265rtppacketizer.hpp

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2020 Filip Klembara (in2core)
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+#ifndef RTC_H265_RTP_PACKETIZER_H
+#define RTC_H265_RTP_PACKETIZER_H
+
+#if RTC_ENABLE_MEDIA
+
+#include "mediahandlerrootelement.hpp"
+#include "h265nalunit.hpp"
+#include "rtppacketizer.hpp"
+
+namespace rtc {
+
+/// RTP packetization of h265 payload
+class RTC_CPP_EXPORT H265RtpPacketizer final : public RtpPacketizer,
+                                               public MediaHandlerRootElement {
+	shared_ptr<H265NalUnits> splitMessage(binary_ptr message);
+	const uint16_t maximumFragmentSize;
+
+public:
+	/// Default clock rate for H265 in RTP
+	inline static const uint32_t defaultClockRate = 90 * 1000;
+
+	/// NAL unit separator
+	enum class Separator {
```

Comment:
This should be shared with `H264RtpPacketizer`.

Reply:
Put it into class NalUnit to fix this comment.
A better refactor would make NalUnit a virtual class then make H264 and H265 implement it. But to make safer and incremental change, I just leave H264 as it is now, making only H265NalUnit inherent from the NalUnit class.